src/Native/libunwind/include/__libunwind_config.h

```diff
@@ -44,8 +44,8 @@
 #    define _LIBUNWIND_CONTEXT_SIZE 76
 #    define _LIBUNWIND_CURSOR_SIZE 83
 #  else
-#    define _LIBUNWIND_CONTEXT_SIZE 58
-#    define _LIBUNWIND_CURSOR_SIZE 65
+#    define _LIBUNWIND_CONTEXT_SIZE 50
```

Comment:
I have changed these sizes for non-amd64 targets when I was modifying libunwind just by adding the delta I expected that was right. I could have made a mistake in that.
@sergign60 I assume you've made these changes because you got build time static asserts, right?